Extension/src/LanguageServer/client.ts

```diff
@@ -745,6 +751,43 @@ class DefaultClient implements Client {
         }
     }
 
+    private promptCompileCommands(params: CompileCommandsPaths) : void {
+        if (this.configuration.Configurations[this.configuration.CurrentConfiguration].compileCommands !== undefined) {
+            return;
+        }
+
+        let showCompileCommandsSelection: PersistentState<boolean> = new PersistentState<boolean>("CPP.showPromptCompileCommands", true);
+        if (!showCompileCommandsSelection.Value) {
+           return;
+        }
+
+        let message: string = "Detected compile_commands.json. Would you like to select a compile_commands.json for this configuration?";
+        let yes: string = "Yes";
+        let later: string = "Later";
```

Comment:
"Later" feels like an affirmative choice.  I think you meant "Ask Me Later".  In the interest of keeping labels short, "Not Now" is another alternative.

Reply:
![screenshot](https://user-images.githubusercontent.com/35574547/41492683-8294dbfc-70b5-11e8-80de-4f947bd5d520.png)
I did it to match VSCode's update message. I do agree that the alternatives you offer sound better.

Reply:
Thanks for the context. They are not asking a question, so "Later" is not an answer.